motionplan/solverFrame.go

```diff
@@ -13,46 +13,44 @@ import (
 // solverFrames are meant to be ephemerally created each time a frame system solution is created, and fulfills the
 // Frame interface so that it can be passed to inverse kinematics.
 type solverFrame struct {
-	name       string
-	fss        frame.FrameSystem
-	movingGeom []string      // List of names of all frames that could move, used for collision detection
-	frames     []frame.Frame // all frames directly between and including solveFrame and goalFrame. Order not important.
-	solveFrame frame.Frame
-	goalFrame  frame.Frame
+	name         string
+	fss          frame.FrameSystem
+	movingFrames []string      // List of names of all frames that could move, used for collision detection
```

Comment:
Could we expand this mildly to have an example: "such as a gripper attached to an arm which is moving relative to World, which would not be in `frames` below"